src/Tribe/App_Shop.php

```diff
@@ -77,11 +79,25 @@ public function add_toolbar_item() {
 		}
 
 		/**
+		 * Registers the plugin assets
+		 */
+		protected function register_assets() {
+			tribe_assets(
+				Tribe__Main::instance(),
+				array(
+					array( 'app-shop-css', 'app-shop.css' ),
+					array( 'app-shop-js', 'app-shop.js' ),
+				),
+				'admin_print_styles-' . $this->admin_page
```

Comment:
Swapping that out for `admin_enqueue_scripts` would cause the styles to be added to every page, which is what Brook was attempting to avoid, I think.